tests/test_preprocessing.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from pyriemann_qiskit.utils.preprocessing import NdRobustScaler
+from sklearn.preprocessing import RobustScaler
+
+
+def test_NdRobustScaler(get_covmats):
```

Comment:
Keep lower case for test names:
```suggestion
def test_ndrobustscaler(get_covmats):
```